mysql-test/mroonga/storage/create/table/table/normalizer/t/normalizer_table.test

```diff
@@ -0,0 +1,51 @@
+# Copyright(C) 2014  Naoya Murakami <naoya@createfield.com>
```

Comment:
The copyright notice is incorrect.

Reply:
fix: [588c2ce](https://github.com/mroonga/mroonga/pull/1018/commits/588c2ceadee85fb93805d22ced7f416c18f973f0)